packages/cmrjs/tests/cmr-utils/test-constructonlineaccessurls.js

```diff
@@ -11,6 +11,14 @@ const constructOnlineAccessUrls = cmrUtils.__get__('constructOnlineAccessUrls');
 
 const sortByURL = (a, b) => a.URL < b.URL;
 
+const endpoint = 'https://endpoint';
+const s3CredentialsEndpointObject = {
+  URL: `${endpoint}/s3credentials`,
+  Description: 'S3 Credentials Endpoint',
+  URLDescription: 'S3 Credentials Endpoint',
```

Comment:
This description should be updated to be more informative.

`api endpoint to retrive temporary credentals valid for same-region direct s3 access.` Or something like that I think.

Reply:
+1